content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx

```diff
@@ -922,9 +932,7 @@ prisma.log.create({
 
 <Admonition type="info">
 
-These _null enums_ do not apply to MongoDB because there the difference between a JSON `null` and a database `NULL` does not exist.
-
-They also do not apply to the `array_contains` operator in all databases because there can only be a JSON `null` within a JSON array. Since there cannot be a database `NULL` within a JSON array, `{ array_contains: null }` is not ambiguous.
+These _null enums_ do not apply to MongoDB because there the difference between a JSON `null` and a database `NULL` does not exist. They also do not apply to the `array_contains` operator in all databases because there can only be a JSON `null` within a JSON array. Since there cannot be a database `NULL` within a JSON array, `{ array_contains: null }` is not ambiguous.
```

Comment:
```suggestion
These _null enums_ do not apply to MongoDB because MongoDB does not differentiate between a JSON `null` and a database `NULL`. They also do not apply to the `array_contains` operator in all databases because there can only be a JSON `null` within a JSON array. Since there cannot be a database `NULL` within a JSON array, `{ array_contains: null }` is not ambiguous.
```